style-guide/src/styles/otkit-icons.module.css

```diff
@@ -25,6 +26,11 @@
   justify-content: center;
 }
 
+.icon {
+  width: icon-size;
+  height: icon-size;
```

Comment:
Can we just set one of the dimensions and rely on aspect ratio preservation? This'll kinda serve as a test and an example (does that make sense?).

Reply:
That would be interesting. Let me try it.